content/issues/2021-07-07-update-center-not-refreshed.md

```diff
@@ -1,15 +1,19 @@
 ---
 title: Update center content not refreshed, recent plugin releases not visible
 date: 2021-07-07T10:00:00-00:00
-resolved: false
-resolvedWhen:
+resolved: true
+resolvedWhen: 2021-07-12T07:02:00-00:00
 # Possible severity levels: down, disrupted, notice
 severity: notice
 affected:
   - updates.jenkins.io
 section: issue
 ---
 
+[update] Issues fixed.
+
+----
+
 [INFRA-3023](https://issues.jenkins.io/browse/INFRA-3023)
 [NOTES](https://hackmd.io/Yg8kpn5HSdyPJI7yVrV_7w)
 
```

Comment:
We should probably remove everything below here, that's not helpful and even misleading.

Reply:
Sorry I merged too quickly. Let me open a new PR.